### PR TITLE
New package: vorta-0.8.10

### LINKED ISSUE #40213: Package request: Vorta
### Package name

vorta

### Package homepage

https://vorta.borgbase.com/

### Description

Vorta is a backup client for macOS and Linux desktops. It integrates the mighty [Borg Backup](https://borgbackup.readthedocs.io/en/stable/) with your favorite desktop environment to protect your data from disk failure, ransomware and theft.

This provides much easier access to most (if not all) borg backup commands. Comparing it to pika backup: pika backup does not (afaik) have any of the extract/mount restore functionality, vorta has way more features than that however.

### Does the requested package meet the package requirements?

System

### Is the requested package released?

Yes

### PR DESCRIPTION
**Pull request by abdeoliveira**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl
  - armv7l
  - armv6l-musl

Closes #40213

### CODE REVIEW

File: srcpkgs/vorta/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+build_style=python3-module
+#Running pytest as root leads 'test_export_fail_unwritable' to fail.
+#'test_ssh_dialog' fails under xbps-src (either 'xbps-src check' or
+#executing pytest into 'xbps-src chroot' enviroment) and I don't know why. 
+#On the other hand, it passes if pytest(ed) chroot(ed) into 
+#the 20221001_glibc release (either as root or user). May be an issue 
+#specific to xbps-src?
+make_check_args="--deselect=tests/test_import_export.py::test_export_fail_unwritable
+ --deselect=tests/test_repo.py::test_ssh_dialog"
+hostmakedepends="python3-setuptools python3-wheel python3-pip"
```
Comment (paper42) [inline srcpkgs/vorta/template:14]: Remove pip, it's only downloading dependencies behind your back and let's you ignore issues with the template.

Comment (abdeoliveira) [reply srcpkgs/vorta/template:14]: Without pip compilation breaks very soon. This is the error: `distutils.errors.DistutilsError: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpoe4d9e4t', '--quiet', 'pip>=10']' returned non-zero exit status 1.`

Do we really need to remove pip? I was partially inspired by borg's template in which `hostmakedepends` include it.

Comment (paper42) [reply srcpkgs/vorta/template:14]: The borg template is wrong and should be fixed, feel free to open a PR.
Vorta has pip in setup_requires in setup.cfg which is wrong, it doesn't need pip. Could you patch it out and send a PR upstream?

Comment (abdeoliveira) [reply srcpkgs/vorta/template:14]: Sure. There will be a learning curve but nothing I couldn't deal with.

Comment (abdeoliveira) [reply srcpkgs/vorta/template:14]: Should I include this conversation l the PRs (to upstream and to borg's)?

Comment (abdeoliveira) [reply srcpkgs/vorta/template:14]: Since I have no technical arguments to support the PRs.

Comment (abdeoliveira) [reply srcpkgs/vorta/template]: Patch included in this PR.

**Comment by abdeoliveira** [comment]
pip remove upstream as per @paper42 suggestion.

File: srcpkgs/vorta/patches/remove-pip-dep.patch [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#PIP is not necessary as per this suggestion:
```
Comment (paper42) [inline srcpkgs/vorta/patches/remove-pip-dep.patch:1]: please import https://github.com/borgbase/vorta/commit/78863544ae532d4c1f8e1ac1319fc613c3cbd9ea.patch instead

File: srcpkgs/vorta/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#'test_export_fail_unwritable' fails as root.
+#'test_ssh_dialog' fails under xbps-src. 
+make_check_args="
+ --deselect=tests/test_import_export.py::test_export_fail_unwritable
+ --deselect=tests/test_repo.py::test_ssh_dialog"
+hostmakedepends="python3-setuptools python3-wheel"
+depends="borg python3-peewee python3-appdirs python3-paramiko
+ python3-SecretStorage python3-psutil python3-PyQt5"
+checkdepends="${depends} dbus python3-pytest python3-pytest-qt
+ python3-pytest-mock python3-six"
```
Comment (paper42) [inline srcpkgs/vorta/template:15]: why is six required here?

Comment (abdeoliveira) [reply srcpkgs/vorta/template:15]: Checking fails complaining the missing module `six`. My guess is that older versions of `paramiko` depended on `six`which is no longer true.  These are the latest error lines :

```
rom paramiko.pkey import PKey
/usr/lib/python3.11/site-packages/paramiko/pkey.py:30: in <module>
    import six
E   ModuleNotFoundError: No module named 'six'
```

File: srcpkgs/vorta/patches/remove-pip-dep.patch (hunk 2 above)
Comment (abdeoliveira) [reply srcpkgs/vorta/patches/remove-pip-dep.patch]: OK.

File: srcpkgs/vorta/template (hunk 3 above)
Comment (paper42) [reply srcpkgs/vorta/template:15]: Then six should be added to paramiko's dependencies, this is clearly not a dependency of vorta.

Comment (abdeoliveira) [reply srcpkgs/vorta/template:15]: I suspect this is the offending PR: https://github.com/void-linux/void-packages/commit/136c139205cc564b00441f00a2734d2533f54469

What should I do? Open a PR to include six back?

Comment (paper42) [reply srcpkgs/vorta/template:15]: That commit is correct, but it removed six from paramiko's dependency tree. It should have always been an explicit dependency. Could you open a PR to add six to paramiko's dependencies?

Comment (abdeoliveira) [reply srcpkgs/vorta/template:15]: Just opened a PR: https://github.com/void-linux/void-packages/pull/42085.

**Comment by abdeoliveira** [comment]
Removed `python3-six` from `checkdepends`

**Pull request closed without merging**